core/trino-main/src/main/java/io/trino/connector/system/jdbc/ColumnJdbcTable.java

```diff
@@ -67,6 +67,7 @@
 import static io.airlift.slice.Slices.utf8Slice;
 import static io.trino.SystemSessionProperties.isOmitDateTimeTypePrecision;
 import static io.trino.connector.system.jdbc.FilterUtil.tablePrefix;
+import static io.trino.connector.system.jdbc.FilterUtil.tablePrefixIfPossible;
```

Comment:
cmt msg

> With the change the more query scenarios are covered by the code path
which does single call to `listTableColumns` instead  ....

it's unclear what additional scenarios we're serving now.

1. exemplify in commit message
2. let's have a test counting metadata calls.